### ISSUE TITLE
Broken Backtick Behaviors 

### ISSUE THREAD

**Issue by cloudRoutine** [ionide/ionide-vscode-fsharp#147]
#### No tooltips for functions with identical signatures

``` fsharp
namespace Yo

module internal ``Momma said knock you out`` =
    let ``i'm gonna knock you out`` _ = "HUUUH!!"
    let samesies _ = "HUUUH!!"
```

![](http://i.imgur.com/HRbYoOR.png)

![](http://i.imgur.com/A102sLJ.png)
#### Broken Autocomplete

On Module

![](http://i.imgur.com/2Wbii30.png)

On Namespace

![](http://i.imgur.com/Sv4VXQ5.png)
### Full Repro Code -

``` fsharp
namespace Yo

module internal ``Momma said knock you out`` =
    let ``i'm gonna knock you out`` _ = "HUUUH!!"
    let samesies _ = "HUUUH!!"

namespace Whatup

module test =
    let stub = () 
    let a1 = Yo.``Momma said knock you out``.``i'm gonna knock you out``
    let a2 = Yo.``Momma said knock you out``.samesies
```

**Comment by Krzysztof-Cieslak** [comment]
Fixed for tooltips, autocomplete still broken.

**Comment by isaacabraham** [comment]
Suggest reopening this one.

![image](https://user-images.githubusercontent.com/1781813/27324030-1a9044f6-55a4-11e7-83a7-9230336e2e51.png)

**Comment by Krzysztof-Cieslak** [comment]
It is open ;P

**Comment by isaacabraham** [comment]
Doh. Saw the closed above. Sorry.

**Comment by Krzysztof-Cieslak** [comment]
Fixed in `3.15.2`
